config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
Are there custom wrappers for the gov.uk toolkit that allow us to write less boilerplate - like how you can generate ones for Bootstrap.